### PR TITLE
Feat/account

### PR DESCRIPTION
**Pull request by umair313**
Implemented User Authentication

### CODE REVIEW

File: SongsApp/settings.py [hunk 1]
```diff
@@ -20,23 +20,30 @@
 # See https://docs.djangoproject.com/en/4.0/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = 'django-insecure-i12@%4+#u=bmk4%xws*8d6(5w&&uoz@^*(gvsu4o+3)cf7v55!'
+SECRET_KEY = config('SECURITY_KEY')
 
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
```
Comment (jawadchughtai6) [inline SongsApp/settings.py:26]: We should eventually have a separate settings file for local dev and prod

File: account/models.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+from django.db import models
+from django.contrib.auth.models import AbstractUser
+# Create your models here.
+
+class User(AbstractUser):
```
Comment (jawadchughtai6) [inline account/models.py:5]: The "following" field will be set later?

File: account/views.py [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+
+from .forms import UserRegisterationForm
+# Create your views here.
+
+def home(request):
+    return render(request, 'account/default.html')
+
+
+def register_user(request):
```
Comment (jawadchughtai6) [inline account/views.py]: I think it is better if we use class based views

**Review by jawadchughtai6 — CHANGES_REQUESTED** [review]

File: account/models.py (hunk 2 above)
Comment (umair313) [reply account/models.py:5]: yes after creating the `Artist`  model.

File: account/views.py (hunk 3 above)
Comment (umair313) [reply account/views.py]: updated

**Pull request merged**